kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/ExpositionIT.java

```diff
@@ -145,6 +144,51 @@ void exposesTwoClusterOverPlainWithSeparatePorts(KafkaCluster cluster) {
         }
     }
 
+    @Test
+    void exposesSingleClusterWithMultipleListeners(KafkaCluster cluster) throws Exception {
```

Comment:
I think it's also a good idea to test multiple SNI listeners. (non-blocking, can add in separate PR if preferred)

https://github.com/k-wall/kroxylicious/pull/53